esp-hal/src/i2c/master/mod.rs

```diff
@@ -647,6 +743,16 @@ impl<'a> I2cFuture<'a> {
             )));
         }
 
+        #[cfg(not(esp32))]
+        if r.scl_st_to().bit_is_set() {
```

Comment:
I know it's a pain, but could we patch the pacs to avoid these cfgs?

Reply:
Again, both of these are not(esp32), so there's nothing to rename these to

Reply:
these are not different but absent on ESP32

Reply:
Oops, sorry for the noise!